App.js

```diff
@@ -1,26 +1,47 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState } from 'react';
+import { StyleSheet, View, ScrollView, ImageBackground, Button } from 'react-native';
+import styled from 'styled-components/native';
+import Weather from './components/Weather';
+import Compass from './components/Compass';
+import Heading from './components/Heading';
+import MyLocation from './components/MyLocation';
 
 const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
-
-const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+	flex: 1;
+	background-color: black;
```

Comment:
Indentation should follow the code guide lines. I'll comment on this here but it's feedback for the whole project.